python/python/glide/async_commands/core.py

```diff
@@ -765,6 +765,37 @@ async def lpop_count(self, key: str, count: int) -> Optional[List[str]]:
             await self._execute_command(RequestType.LPop, [key, str(count)]),
         )
 
+    async def blpop(self, keys: List[str], timeout: float) -> Optional[List[str]]:
+        """
+        Pops an element from the head of the first list that is non-empty, with the given keys being checked in the
+        order that they are given.
+        Blocks the connection when there are no elements to pop from any of the given lists.
+
+        See https://redis.io/commands/blpop/ for details.
+
+        Note: BLPOP is a client blocking command, see
+        https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands for more details and best
+        practices.
+
+        Args:
+            keys (List[str]): The keys of the lists to pop from.
+            timeout (float): The number of seconds to wait for a blocking BLPOP operation to complete. A
```

Comment:
```suggestion
            timeout (float): The number of seconds to wait for a blocking operation to complete. A
```
San proposed to remove command name there - I'll do for all blocking commands I did in java